mod_form.php

```diff
@@ -101,11 +101,11 @@ public function definition() {
 
         $mform->addElement('header', 'calendaroptions', get_string('calendaroptions', 'facetoface'));
 
-        $calendaroptions = array(
+        $calendaroptions = [
             F2F_CAL_NONE   => get_string('none'),
-            F2F_CAL_COURSE => get_string('course'),
-            F2F_CAL_SITE   => get_string('site')
-        );
+            F2F_CAL_COURSE => get_string('course')
+        ] + (has_capability('mod/facetoface:createsitewideevent', $this->context) ? [F2F_CAL_SITE   => get_string('site')] : []);
```

Comment:
This is a bit hard to read, maybe split out the ternary to a variable?

Reply:
Actually, it would be clearer to read as a basic if condition after the array.

e.g.
```
array = [ 2 options ] 
if (condition) {
array[key] = value
}
```

Reply:
Thanks @keevan - updated the patch.